tests/integration/components/x-toggle/component-test.js

```diff
@@ -0,0 +1,97 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
Yes this is an important test. The number of times `false` puts a spanner in the works is high. :)